app/src/main/java/com/android/partagix/ui/BottomNavigationMenu.kt

```diff
@@ -16,11 +17,13 @@ import com.android.partagix.ui.navigation.TopLevelDestination
 @Composable
 fun BottomNavigationBar(
     selectedDestination: String,
-    navigateToTopLevelDestination: (TopLevelDestination) -> Unit
+    navigateToTopLevelDestination: (TopLevelDestination) -> Unit,
+    modifier: Modifier
```

Comment:
It's a good practice to add a default value for modifier.
- Original:
```
modifier: Modifier
```
- Proposed modification:
```
modifier: Modifier = Modifier
```